docs/pages/deploy-a-cluster/helm-deployments/aws.mdx

```diff
@@ -121,65 +115,29 @@ communications and allow external clients to connect.
 
 ### Determining an approach
 
-There are three supported options when using AWS. You must choose only one of
-these options:
-
-#### Using `cert-manager`
-
-You can use `cert-manager` to provision and automatically renew TLS credentials
-by completing ACME challenges via Let's Encrypt. We recommend this approach if
-you require CLI access to web applications using client certificates via
-Teleport Application Access.
-
-#### Using AWS Certificate Manager
-
-You can use AWS Certificate Manager to handle TLS termination with AWS-managed
-certificates.
+There are three supported options on AWS (choose one):
```

Comment:
I do get where you are coming from, maybe we can find a middle ground.  Would it be possible for choosing an approach to be summarized in a visual (a table of supported/unsupported features)? Does that seem too complicated?

(opinion) As a user, I'm use to having the recommended approach default, front and center.  It seems like we have a soft recommendation for cert-manager.  I wonder if given the number of drawbacks with AWS Certificate Manager, we shouldn't be more strongly encouraging users to pursue cert-manager if they are at all on the fence.  This probably explains why I'd more naturally lean towards promoting a preferred route while still providing optional paths and hiding unrelated information behind tabs/pages.

Reply:
I'm definitely all for reducing the number of options in order to simplify this page. @webvictim @hugoShaka What do you think of recommending cert-manager here?

Reply:
I don't think that's doable. We have 2 typical use cases:
- cert-manager + Teleport terminating TLS: the most versatile (better for us), more complex as it involves more maintenance on the user side. Also, might provide less security guarantees than ACM, requires to grant powerful DNS rights to a workload, not the best from a compliance POV.
- ACM + an Amazon-managed LB Terminating TLS: less portable (more work and complexity for us), worse UX because some features require TLS Termination, can't run TLS Multiplexing (yet). But it's simpler from a user pov, more secure, compliant, and likely already integrated in the customer stack.

Cert-manager-only is easier for us but worse for the users. A lot of users are using ACM because it's better for them and I don't think we can afford to remove it.

Reply:
@evanfreed Just checking to see if you have any thoughts on this thread.

Reply:
It's somewhat easier to use ACM now as Teleport 13 added support for multiplexing behind ALBs, which also opens up the ability to use a Kubernetes Ingress.

I'm working on a partial docs rewrite for this here: https://github.com/gravitational/teleport/pull/25815

Reply:
> I don't think we can afford to remove it.

Plan wasn't to remove it, just to have different tabs for the differing cert options.  That way text is only displayed based on the users needs.

Sorry for the delay this fell off my radar. Given the discussion and my late response I'll just revert this part back to how it was and simplify the changes a bit.